### PR TITLE
Add several metrics in compute

### LINKED ISSUE #287: Compute : Add the possibility to apply a math formula on several metrics

### PR DESCRIPTION
**Pull request by mbarbet**
fix #287

### CODE REVIEW

File: src/contributors/ComputeContributor.ts [hunk 1]
```diff
@@ -54,28 +51,29 @@ export class ComputeContributor extends Contributor {
...
     }
 
-    public setData(data: ComputationResponse): any {
-        const result = data.value;
-        let resultValue = result;
-        if (this.process && this.process.trim().length > 0) {
-            resultValue = eval(this.process);
-        }
+    public setData(data: Array<ComputationResponse>): any {
+        const m = data.map(d => d.value);
```
Comment (sebbousquet) [inline src/contributors/ComputeContributor.ts]: `m` is never used

Comment (mbarbet) [reply src/contributors/ComputeContributor.ts]: I invite you to read the json schema, m is used in the function inside the eva

Comment (sebbousquet) [reply src/contributors/ComputeContributor.ts]: Ok, I see it

File: src/contributors/ComputeContributor.ts [hunk 2]
```diff
@@ -18,25 +18,22 @@
...
 /**
  * This contributor computes a metric on the given field, given the filters
  */
 export class ComputeContributor extends Contributor {
 
-    /** Field on which the metric will be computed*/
-    public field: string = this.getConfigValue('field');
-    /** Field on which the metric will be computed*/
-    public metric: string = this.getConfigValue('metric');
+    /** Array of which the metric will be computed*/
```
Comment (MohamedHamouGisaia) [inline src/contributors/ComputeContributor.ts]: `which metrics` instead of `which the metric`

**Pull request merged**